tests/pytests/unit/modules/test_pacmanpkg.py

```diff
@@ -63,106 +66,29 @@ def test_list_pkgs_as_list():
     assert stringifymock.call_count == 0
 
 
-def test_list_pkgs_no_context():
+def test_pacman_install_sysupgrade_flag():
     """
-    Test if it list the packages currently installed in a dict
+    Test if the pacman.install function appends the '-u' flag only when sysupgrade is True
     """
-    cmdmock = MagicMock(return_value="A 1.0\nB 2.0")
-    sortmock = MagicMock()
-    stringifymock = MagicMock()
-    mock_ret = {"A": ["1.0"], "B": ["2.0"]}
+    mock_parse_targets = MagicMock(return_value=({"somepkg": None}, "repository"))
+    mock_has_scope = MagicMock(return_value=False)
+    mock_list_pkgs = MagicMock(return_value={"somepkg": "1.0"})
+    mock_run_all = MagicMock(return_value={"retcode": 0, "stderr": ""})
+
     with patch.dict(
         pacman.__salt__,
         {
-            "cmd.run": cmdmock,
-            "pkg_resource.add_pkg": lambda pkgs, name, version: pkgs.setdefault(
-                name, []
-            ).append(version),
-            "pkg_resource.sort_pkglist": sortmock,
-            "pkg_resource.stringify": stringifymock,
+            "cmd.run_all": mock_run_all,
+            "pkg_resource.parse_targets": mock_parse_targets,
+            "config.get": MagicMock(return_value=True),
         },
-    ), patch.object(pacman, "_list_pkgs_from_context") as list_pkgs_context_mock:
-        assert pacman.list_pkgs() == mock_ret
-
-        pkgs = pacman.list_pkgs(versions_as_list=True, use_context=False)
-        list_pkgs_context_mock.assert_not_called()
-        list_pkgs_context_mock.reset_mock()
-
-        pkgs = pacman.list_pkgs(versions_as_list=True, use_context=False)
-        list_pkgs_context_mock.assert_not_called()
-        list_pkgs_context_mock.reset_mock()
-
-
-def test_group_list():
```

Comment:
Did these tests get moved somewhere else?

Reply:
Uh oh.  That was unintentional.  Definitely didnt mean to remove tests.